third_party/move/move-compiler-v2/tests/more-v1/parser/invalid_call_lhs_value2.exp

```diff
@@ -1,10 +1,13 @@
 
 Diagnostics:
-error: unexpected token
-  ┌─ tests/more-v1/parser/invalid_call_lhs_value2.move:3:10
+error: expected `|()|_` but found a value of type `integer`
+  ┌─ tests/more-v1/parser/invalid_call_lhs_value2.move:3:9
   │
 3 │         5();
-  │          ^
-  │          │
-  │          Unexpected '('
-  │          Expected ';'
+  │         ^
+
+error: expected `|(integer, integer)|_` but found a value of type `integer`
```

Comment:
the description of expected is likely not intuitive to the user: should it not be `|integer, integer|` instead of `|(integer, integer)|_`?

Reply:
The function doesn't exist.  We don't know the return type here, so it is written as `_`.

We can improve this, but it shouldn't be in this PR, as it will change all test outputs.

Filed https://github.com/aptos-labs/aptos-core/issues/15415